gateway-rsp-test-gateway-swagger.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I am unfamiliar with Swagger. Can you give me a brief explanation of what we will use this file for please? If this is Infrastructure as Code (IaC) then can you drop this into the `iac` folder instead.